internal/pkg/server/fleet.go

```diff
@@ -406,22 +405,24 @@ func (f *Fleet) runServer(ctx context.Context, cfg *config.Config) (err error) {
 func (f *Fleet) runSubsystems(ctx context.Context, cfg *config.Config, g *errgroup.Group, bulker bulk.Bulk, tracer *apm.Tracer) (err error) {
 	esCli := bulker.Client()
 
-	// Check version compatibility with Elasticsearch
-	remoteVersion, err := ver.CheckCompatibility(ctx, esCli, f.bi.Version)
-	if err != nil {
-		if len(remoteVersion) != 0 {
-			return fmt.Errorf("failed version compatibility check with elasticsearch (Agent: %s, Elasticsearch: %s): %w",
-				f.bi.Version, remoteVersion, err)
+	if !f.standAlone {
```

Comment:
Can we add a comment that explains why the version check and migrations are disabled for standalone mode (for now)?

Reply:
Added.